3rdParty/velocypack/src/Validator.cpp

```diff
@@ -220,7 +216,35 @@ bool Validator::validate(uint8_t const* ptr, std::size_t length, bool isSubPart)
 
   // common validation that must happen for all types
   validateSliceLength(ptr, length, isSubPart);
-  return true;
+}
+
+void Validator::validateTagged(uint8_t const* ptr, std::size_t length) {
+  uint8_t head = *ptr;
+     
+  do {
+    // looping here to skip over nested tags.
+    if (head == 0xee) {
+      // 1 byte tag type
+      // the actual Slice (without tag) must be at least one byte long
+      validateBufferLength(1 + 1 + 1, length, true);
+      VELOCYPACK_ASSERT(length > 2);
+      ptr += 2;
+      length -= 2;
+    } else if (head == 0xef) {
+      // 8 bytes tag type
+      // the actual Slice (without tag) must be at least one byte long
+      validateBufferLength(1 + 8 + 1, length, true);
+      VELOCYPACK_ASSERT(length > 9);
+      ptr += 9;
+      length -= 9;
+    } else {
+      throw Exception(Exception::NotImplemented);
```

Comment:
Here again, what is not implemented would be a helpful error message addition for debugging